src/attributes/codegen.md

```diff
@@ -230,6 +230,27 @@ Feature        | Implicitly Enables | Feature Name
 `tme`          |                | FEAT_TME --- Transactional Memory Extension
 `vh`           |                | FEAT_VHE --- Virtualization Host Extensions
 
+#### `loongarch`
+
+r[attributes.codegen.target_feature.loongarch]
+
+This platform requires that `#[target_feature]` is only applied to [`unsafe`
+functions][unsafe function].
+
+Further documentation on these features can be found in the [LoongArch Reference
+Manual], or elsewhere on [LoongArch GitHub Account].
+
+[LoongArch Reference Manual]: https://github.com/loongson/LoongArch-Documentation
+[LoongArch GitHub Account]: https://github.com/loongson
```

Comment:
Is there documentation that is more alive than these links? The documentation link looks like it is archived, and I don't know what else I would look at on the GitHub account. Is there a new location where the documentation lives? Is there a way to deep-link to the individual features (like wasm does)?

Reply:
The currently available English documentation is at https://github.com/loongson/LoongArch-Documentation. If a better option becomes available in the future, I will update it. Since not all target features have fragment identifiers, they point to the nearest location.

Reply:
Can you say more about what's going on with the public documentation? Do you know why it is archived? Is there an actively maintained copy of the Chinese version? Are there any intentions to revive the English version, or is it moving somewhere else?

Reply:
Of course! Based on the latest info I've received, the [LoongArch-Documentation](https://github.com/loongson/LoongArch-Documentation) is not permanently archived and is still maintained, with updates made as needed. It's currently the only source for the referenced information in English. The Chinese version can be downloaded from: [龙芯架构参考手册卷一-r1p11.pdf](https://loongson.cn/uploads/images/2025032109191292796.%E9%BE%99%E6%9E%B6%E6%9E%84%E5%8F%82%E8%80%83%E6%89%8B%E5%86%8C%E5%8D%B7%E4%B8%80_r1p11.pdf). There might be a new English version released in the future, though there's no clear timeline for that yet. The potential update is more about improving the formatting rather than addressing any issues with the existing content.

Reply:
I have pushed updates to the `LoongArch-Documentation`, adding anchors for each CPU feature. This documentation now supports deep linking to individual features.